spring-cloud-azure-appconfiguration-config/src/test/java/com/microsoft/azure/spring/cloud/config/stores/ConfigStoreTest.java

```diff
@@ -26,5 +27,21 @@ public void invalidEndpoint() {
         configStore.validateAndInit();
         fail();
     }
+    
+    @Test
+    public void getLabelsTest() {
+        ConfigStore configStore = new ConfigStore();
+        assertEquals(configStore.getLabels()[0], "\0");
+        
+        configStore.setLabel("dev");
+        assertEquals(configStore.getLabels()[0], "dev");
+        
+        configStore.setLabel("dev,test");
+        assertEquals(configStore.getLabels()[0], "test");
+        assertEquals(configStore.getLabels()[1], "dev");
+        
+        configStore.setLabel(",");
+        assertEquals(configStore.getLabels()[0], "\0");
+    }
```

Comment:
Can we breakdown this test in others 4 saying what the expected behavior for each one? This way, if some tests break we can see easily which behavior it was impacted for some change in the future.

For instance: `shouldStartLabelAsZeroCharacter`,  `shouldReplaceLabelBySingleValue`, `shouldSplitLabelByComma`....